packages/sdk/src/types/runtime/actor.ts

```diff
@@ -437,6 +439,59 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
         return null;
     }
 
+    /**
+     * Starts playing a preloaded sound.
+     * @param soundAssetId Name of sound asset preloaded using AssetManager.
+     * @param options Adjustments to pitch and volume, and other characteristics.
+     * @param startTimeOffset How many seconds to offset into the sound
+     */
+    public playSound(

```

Comment:
>playSound [](start = 11, length = 9)

This API should follow the pattern of enableText, enableRigidBody, etc.: returns void, but sets an actor property for retrieval afterward.

Reply:
No, because it doesn't match a regular component structure.
1. Sounds (and particle effects) auto-expire (if they are non-looping, of course). The expiration is automatically dealt with by clients, and therefore the node-side can just ignore the results. There's not a good use case for listening to when sounds start/stop, so it would just be unnecessary overhead to track this on node side.
2. There can be multiple simultaneously active sound instances on a single actor